app/stores.py

```diff
@@ -0,0 +1,25 @@
+from app.data_jason import load_json
+import dataclasses
+from typing import List
+
+
+@dataclasses.dataclass
+class Stores:
```

Comment:
Better to name class in single
**Store**

Reply:
do it

Reply:
Better to name class in single
Store